internal/system/agent/executor/compose.go

```diff
@@ -102,15 +101,14 @@ func RunDockerComposeCommand(service string, dockerComposeService string) {
 	cmdName := "docker-compose"
 
 	// Retry the fetching of the docker-compose.yml from the Github edgexfoundry repository.
-	// fmt.Sprintf("Pulling latest compose file from Github edgexfoundry repository...")
 	err = Do(func(attempt int) (bool, error) {
 		var err error
 		cmdDir, err = FetchDockerComposeYamlAndPath()
 		// Try 5 times
 		return attempt < 5, err
 	})
 	if err != nil {
-		log.Fatalln("Unable to pull the latest compose file from Github edgexfoundry repository.:", err)
+		log.Printf("Unable to pull the latest compose file from Github edgexfoundry repository: %s", err)
```

Comment:
Any reason we're not using the same LoggingClient as the other services?

Reply:
Yep, much as I would've preferred to use the _same_ LoggingClient as the other services, I ran into a circular-dependency issue. In fact, I alluded to that in the following point in the description atop this PR:
`The only reason why Go's built-in (overly-rudimentary) logging (viz. log.Fatal()) was used in the first place has to do with how the package structuring that was not allowing the import of the clients/logging package.`

Reply:
This is the specific error seen on trying to use the same LoggingClient as the other services:
`CGO_ENABLED=0 go build -ldflags "-X github.com/edgexfoundry/edgex-go.Version=0.7.0" -o cmd/sys-mgmt-agent/sys-mgmt-agent ./cmd/sys-mgmt-agent
import cycle not allowed
package github.com/edgexfoundry/edgex-go/cmd/sys-mgmt-agent
        imports github.com/edgexfoundry/edgex-go/internal/system/agent
        imports github.com/edgexfoundry/edgex-go/internal/system/agent/executor
        imports github.com/edgexfoundry/edgex-go/internal/system/agent
make: *** [cmd/sys-mgmt-agent/sys-mgmt-agent] Error 1`

Reply:
I just refactored the files in the executor package to use the EdgeX LoggingClient. I'm not seeing a circular import reference on build. You can find the work here.
https://github.com/tsconn23/edgex-go/tree/issue_763_tsc

Reply:
**TODO:** Need to revisit in near-future (for standardized logging across services) 👍

Reply:
I don't understand why this is a TODO when the code is in the branch referred to above and simply needs to be pulled in, but so be it.

Assigned to you https://github.com/edgexfoundry/edgex-go/issues/780